src/styles/main.scss

```diff
@@ -1,6 +1,8 @@
 @import 'shared-globals';
 @import '@angular/material/theming';
 
+@include mat-core(mat-typography-config($font-family: $font-family));
```

Comment:
mat-core is already included in this file on line 69, please update the parameters of that invocation.

Reply:
oops

Reply:
done